src/interpreter.ts

```diff
@@ -102,21 +106,23 @@ export async function readQRCodeFromImageFile(
   )
 }
 
-export default async function interpretFile(
-  interpretFileParams: InterpretFileParams
-) {
-  const { election, ballotImagePath, cvrCallback } = interpretFileParams
+export default class SummaryBallotInterpreter implements Interpreter {
+  public constructor() {}
```

Comment:
Unnecessary constructor?

Reply:
it was complaining about the constructor needing an argument when I didn't have it... is it possible the default constructor needs an argument?

Reply:
https://github.com/votingworks/module-scan/pull/55